app/src/main/java/de/westnordost/streetcomplete/map/MainFragment.kt

```diff
@@ -657,7 +659,7 @@ class MainFragment : Fragment(R.layout.fragment_main),
     }
 
     private fun onClickLocationPointer() {
-        setIsFollowingPosition(true)
+        mapFragment?.followPosition(true)
```

Comment:
Not sure what you are trying to do, but this looks like a hacky approach. See my other review comment.

Reply:
Trying to center the screen on your current position, but not lock on (that's reserved for the gps button).

I think it looks hacky because of the name of the function. It's `followPosition`, but really it just moves the view to be centered on your current position. Previously, it would only do that if the map fragment was set to following position (and this was a protected function; I made it public to enable calling it from the MainFragment); I added a switch to override that, to force a center even if it's not following.

I intentionally implemented this in few lines of code, but I'll talk more about that in reply to the review comment.

Reply:
Ah, I thought that tapping on the location pointer pin (etc) should also trigger the follow-mode.

Reply:
I thought it felt confusing/awkward to have the gps button disappear when tapping on the location pin. Also (much less important), I think sometimes it's nice to be able to return to your location without locking on.

Reply:
Note: *Currently* this happens when tapping on both the location pin (when I'm off screen) and marker (when on screen).

For the pin, it very much feels like the right thing to do / what I expect to happen.
For the marker, I think it feels a bit weird. I'm not sure it should do anything at all, actually.

Reply:
You could remove that functionality from the marker.

Reply:
Done (b8ad4d9). Leaving this conversation open since the function name is not updated yet.

Reply:
FWIW, upon reflection, I think the reason tapping the marker feels weird is that it's displayed flatly on the map, like the background, which is not tap-able; whereas the quest pins, location pin, and gps button all display as hovering above the map. So, making the marker tap-able breaks consistency. I think this is probably also my main problem with the live app's behavior.